homeassistant/components/ecobee/services.yaml

```diff
@@ -196,3 +196,104 @@ set_occupancy_modes:
       description: Enable Follow Me mode.
       selector:
         boolean:
+
+set_ventilator_mode:
```

Comment:
Home Assistant already has `fan_mode` support for climate entities. This should be used instead of creating its own services.

See: <https://developers.home-assistant.io/docs/core/entity/climate>